tests/TestCase/Database/Schema/MysqlSchemaTest.php

```diff
@@ -85,36 +85,36 @@ public static function convertColumnProvider()
                 ['type' => 'boolean', 'length' => null],
             ],
             [
-                'TINYINT(2)',
-                ['type' => 'tinyinteger', 'length' => 2, 'unsigned' => false],
+                'TINYINT(1) UNSIGNED',
+                ['type' => 'boolean', 'length' => null],
             ],
             [
                 'TINYINT(3)',
-                ['type' => 'tinyinteger', 'length' => 3, 'unsigned' => false],
+                ['type' => 'tinyinteger', 'length' => null, 'unsigned' => false],
```

Comment:
Will dropping display lengths impact older versions of MySQL where it wasn't just decorative?

Reply:
It should not. They were onlu used with the `zerofill` attribute which fixtures/schemas don't support.

Reply:
https://dev.mysql.com/doc/refman/8.0/en/numeric-type-syntax.html the top paragraph has the notes on this.

Reply:
Ok. It looks like integer widths were only for display purposes as [far back as 5.6 ](https://dev.mysql.com/doc/refman/5.6/en/numeric-type-attributes.html). With that in mind, ignoring display widths seems reasonable as it doesn't impact storage types.

Reply:
To avoid surprising people in a bug fix release we might want to move this to 4.1.0.

Reply:
Sounds good. I started it in 4.0 thinking it would be simpler, but with these changes 4.1 makes sense.